### PR TITLE
feat: Give necessary permissons to generated api

### PR DESCRIPTION
**Pull request by karelianpie**
## Motivation and Context

Allow discord bot api users to access/update data with their api key

For example, in order to read a profile

`{"users":{"circle":{"api_keys":{"_and":[{"hash":{"_eq":"X-Hasura-Api-Key-Hash"}},{"read_member_profiles":{"_eq":true}}]}}}}`

## Description

Add all permissions to discord bot api users

## Reviewers

@topocount

### CODE REVIEW

**Review by topocount — COMMENTED** [review]
@karelianpie please clean this commit history up and rebase it so it's reviewable. It's very difficult to follow the intent here with all the extra commits.

File: src/pages/DiscordPage/DiscordPage.tsx [hunk 1]
```diff
@@ -59,6 +59,16 @@ export const DiscordPage = () => {
...
+        read_circle: true,
+        update_circle: true,
+        read_nominees: true,
+        create_vouches: true,
+        read_pending_token_gifts: true,
+        update_pending_token_gifts: true,
+        read_member_profiles: true,
+        read_epochs: true,
+        read_contributions: true,
+        create_contributions: true,
```
Comment (topocount) [inline src/pages/DiscordPage/DiscordPage.tsx:71]: Be sure to add the `manage_users` permission here.

**Comment by karelianpie** [comment]
> @karelianpie please clean this commit history up and rebase it so it's reviewable. It's very difficult to follow the intent here with all the extra commits.

@topocount can you please rebase your branch [hasura-api-actions](https://github.com/coordinape/coordinape/tree/hasura-api-actions) with `main` --

File: src/pages/DiscordPage/DiscordPage.tsx (hunk 1 above)
Comment (karelianpie) [reply src/pages/DiscordPage/DiscordPage.tsx:71]: Oh, totally overlooked that one. Also added `read_discord`, not totally sure it'll be necessary though.

https://github.com/coordinape/coordinape/pull/1955/commits/97d2b018e0c2b864f568539b8f09ea752f0bfc8a

**Pull request closed without merging**